perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -20,6 +20,7 @@ use FixMyStreet::App::Form::Waste::Garden::Cancel;
 use FixMyStreet::App::Form::Waste::Garden::Renew;
 use FixMyStreet::App::Form::Waste::Garden::Sacks;
 use FixMyStreet::App::Form::Waste::Garden::Sacks::Renew;
+use FixMyStreet::App::Form::Waste::Garden::Sacks::Modify; # Getting more
```

Comment:
Is there a reason this could not be ::Purchase which would make the purpose clearer?